Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -253,35 +255,124 @@ def is_requires_security_reviewer(pr_files: list[str]) -> bool:
     return False
 
 
-def is_tim_content(pr_files: list[str]) -> bool:
+def check_if_pack_or_integration_is_feed(content_object: dict) -> bool:
     """
-    This is where the actual search for feed:True or relevant tags or categories are being searched
-    according to the login in is_tim_reviewer_needed
+    Checks whether a given object (graph object) is a feed or related to TIM
 
     Arguments:
-    - pr_files: List[str] The list of files changed in the Pull Request.
+        - `content_object`: ``dict``: Content object from the graph
 
-    Returns: returns True or False if tim reviewer needed
+    Returns: `bool` whether the content object is a feed or has the relevant tags/categories
     """
-    integrations_checked = []
-    for file in pr_files:
-        if 'CONTRIBUTORS.json' in file:
-            continue
-        integration = BaseContent.from_path(CONTENT_PATH / file)
-        if not isinstance(integration, Integration) or integration.path in integrations_checked:
-            continue
-        integrations_checked.append(integration.path)
-        if integration.is_feed:
+    print(f'check_if_pack_or_integration_is_feed - content object {content_object}')
+    if isinstance(content_object, Integration):
+        print(f'check_if_pack_or_integration_is_feed - content object {content_object}')
+        if content_object.is_feed:
             return True
-        pack = integration.in_pack
+    try:
+        pack = content_object.in_pack  # type: ignore
+        print(f' check_if_pack_or_integration_is_feed the pack is: {pack} ')
         tags = pack.tags
         categories = pack.categories
         if TIM_TAGS in tags or TIM_CATEGORIES in categories:
             return True
+    except Exception as er:
+        print(f"The pack is not TIM: {er}")
```

Comment:
What exceptions this is intended to catch?

Reply:
the BaseContent.from_path(CONTENT_PATH / file) sometimes returns "not found" for example for images or for items that are not formatted correctly (like incident_field or incident_type missing a mandatory key) 

when not found - content_object.in_pack returns an error. 
This is what I want to catch.